Library/Homebrew/completions/fish.erb

```diff
@@ -111,15 +111,21 @@ function __fish_brew_suggest_formulae_all -d 'Lists all available formulae with
     if test -f "$__brew_cache_path/descriptions.json"
         __fish_brew_ruby_parse_json "$__brew_cache_path/descriptions.json" \
             '.each{ |k, v| puts([k, v].reject(&:nil?).join("\t")) }'
+    else if test -n "$HOMEBREW_NO_INSTALL_FROM_API"
+        command ls -1 (brew --prefix)/Library/Taps/homebrew/homebrew-core
```

Comment:
I'm not sure it's worth having this at all, personally. If there's something similar elsewhere: let's remove it.

Reply:
Logic taken from https://github.com/Homebrew/brew/blob/838ff48e7586c637a6265025959acfb79ccd109f/Library/Homebrew/cmd/formulae.sh#L10-L21
Agree, will need to take out here